packages/dashboards/CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# How To Contribute
```

Comment:
We already have a contrib file on the top level.

Reply:
Adding to the local `.gitignore`